superset-frontend/src/views/components/RightMenu.tsx

```diff
@@ -539,25 +539,38 @@ const RightMenu = ({
         )}
       </Menu>
       {navbarRight.documentation_url && (
-        <StyledAnchor
-          href={navbarRight.documentation_url}
-          target="_blank"
-          rel="noreferrer"
-          title={t('Documentation')}
-        >
-          <i className="fa fa-question" />
-          &nbsp;
-        </StyledAnchor>
+        <>
+          <StyledAnchor
+            href={navbarRight.documentation_url}
+            target="_blank"
+            rel="noreferrer"
+            title={navbarRight.documentation_text || t('Documentation')}
```

Comment:
1. Should we translate `documentation_text`? If not, we probably don't want to translate `bug_report_text` in line 564?
2. Are you sure that `t()` will work correctly with dynamic expression like here (text depends on result of `||` operator). Shouldn't we use `t('%s', navbarRight.documentation_text || 'Documentation')` instead? That's actually more of a question than suggestion, I'm not sure which style is preferred here

Reply:
Oof, the `bug_report_text` translation was a complete mistake from me! Will fix

Reply:
Regarding question 2, I think the pattern in `documentation_text` is correct, i.e. we want to use the string from the config as-is, and only fall back to the translated text if it's undefined.